backend/controllers/patientController.js

```diff
@@ -0,0 +1,44 @@
+const Patient = require('../models/patientModel')
+const mongoose = require('mongoose')
+
+
+const createPatient = async (req, res) => {
+    const {imie, gatunek, rasa, id_wlasciciela} = req.body
+
+    try {
+        const patient = await Patient.create({imie, gatunek, rasa, id_wlasciciela})
+        res.status(200).json(patient)
+    } catch(error) {
+        res.status(400).json({error: error.message})
+    }
+
+    
+}
+
+
+const getPatients = async(req, res) => {
+    const patients = await Patient.find({}).sort({createdAt: -1})
+
+    res.status(200).json(patients)
+}
+
+
+const getPatient = async(req, res) => {
+    const{ id } = req.params
+
+
```

Comment:
Please remove the blank lines. Keep at maximum one blank line for each section of code